comms/discovery/db/queries/get_chat_messages.go

```diff
@@ -13,7 +13,9 @@ import (
 
 // Get a chat message
 const chatMessage = `
-select message_id, chat_id, user_id, created_at, ciphertext from chat_message where chat_id = $1 and message_id = $2
+SELECT chat_message.message_id, chat_message.chat_id, chat_message.user_id, chat_message.created_at, 
```

Comment:
Does this need is_plaintext or anything?

Reply:
afaict this is only used in `validateChatReact` in `validator.go` so i don't think so